firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -123,6 +127,11 @@ uint8_t moveStraight(int motor[], int startCol, int startRow, int endCol, int en
 
   numSteps = unitSpaces * stepsPerUnitSpace;
 
+  numEighthStepsX = 8;
```

Comment:
Can you make a comment explaining that you're setting the scales to 8 because this function always moves in whole steps?

Reply:
Pls also comment that you set the sign based on the direction

Reply:
should be numEighthSteps, not munEightSteps.

Remove: "moved at a time"